docs/source/algorithms/Stitch-v1.rst

```diff
@@ -0,0 +1,94 @@
+
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm performs stitching of multiple 2D workspaces. Stitching is the operation of scaling one workspace to match the scale of another one in the overlap region and the combination of the workspaces after scale matching.
+For each pair of adjacent workspaces in the input, it will find the overlap, interpolate one to another in the overlap region, make the ratio and find the scaling factor.
+Then each workspace will be scaled with that factor, and finally all the workspaces will be combined.
+
+Inputs
+------
+
+InputWorkspaces
+###############
+
+The list of input workspace or workspace group names. The workspace groups will be flattened to a list of workspaces.
+Note that because of this type of input that relies on WorkspaceGroups, this algorithm is currently supported only for named workspaces present on ADS.
```

Comment:
I am a little confused by this sentence. Do you mean here something equivalent to: "This algorithm is currently supported only for named workspaces present in the ADS. Note that because of this, for the input that contains WorkspaceGroups, all workspaces contained in a group must be named and present in the ADS" ?

Reply:
No, it's due to the fact that it relies on WorkspaceGroups, it can support only named workspaces.